client/my-sites/plugins/marketplace/marketplace-plugin-setup-status/index.tsx

```diff
@@ -1,40 +1,106 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { ThemeProvider } from 'emotion-theming';
 import { ProgressBar } from '@automattic/components';
 import styled from '@emotion/styled';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import page from 'page';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import theme from 'calypso/my-sites/plugins/marketplace/theme';
 import Masterbar from 'calypso/layout/masterbar/masterbar';
-import { getStatusForPlugin } from 'calypso/state/plugins/installed/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
 import QueryJetpackPlugins from 'calypso/components/data/query-jetpack-plugins';
-
+import { initiateThemeTransfer } from 'calypso/state/themes/actions';
+import { getAutomatedTransferStatus } from 'calypso/state/automated-transfer/selectors';
 /**
  * Style dependencies
  */
 import 'calypso/my-sites/plugins/marketplace/marketplace-plugin-setup-status/style.scss';
+import { getPurchaseFlowState } from 'calypso/state/plugins/marketplace/selectors';
 
 const StyledProgressBar = styled( ProgressBar )`
 	margin: 20px 0px;
 `;
 
-function WrappedMarketplacePluginSetup( {
-	pluginSlug = 'wordpress-seo',
-}: {
-	pluginSlug?: string;
-} ): JSX.Element {
+// Stages
+
+function WrappedMarketplacePluginSetup(): JSX.Element {
+	const translate = useTranslate();
+
+	const STAGE_1 = translate( 'Installing plugin' );
+	const STAGE_2 = translate( 'Activating plugin' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Activating Plugin' )` *ES Score: 9*